spec/requests/api/automation_requests_spec.rb

```diff
@@ -192,4 +192,40 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  context 'Tasks subcollection' do
+    let(:automation_request) { FactoryGirl.create(:automation_request) }
+
+    it 'redirects to request_tasks subcollection' do
+      task = FactoryGirl.create(:miq_request_task, :miq_request_id => automation_request.id)
+      api_basic_authorize
+
+      run_get("#{automation_requests_url(automation_request.id)}/tasks")
+
+      expect(response).to have_http_status(:moved_permanently)
+      expect(response.redirect?).to be_truthy
+      expect(response.redirect_url).to include("#{automation_requests_url(automation_request.id)}/request_tasks")
+
+      run_get response.redirect_url, :expand => :resources
```

Comment:
Although I like the idea of testing the whole flow of being redirected in this test, on the flip side this could also be a test duplication. I think we could assume that this subsequent request will do the right thing, especially if it is tested elsewhere (assuming it is). So I'm not sure this extra check is necessary. WDYT?